scripts/build-wvr-components.js

```diff
@@ -17,9 +19,10 @@ const dirPath = `dist/bundle/${dirName}`;
   ];
 
   fs.ensureDir(dirPath);
+  fs.ensureDir(latestPath);
 
   await concat(files, `${dirPath}/weaver-components.js`);
+  await concat(files, `${latestPath}/weaver-components.js`);
   fs.copy(`${dirPath}/weaver-components.js`, "dist/docs/usage/weaver-components.js");
```

Comment:
Check if the `fs.copy` command already ensures dir. If so, then line 22 is not needed.